packaging/debian/rules

```diff
@@ -20,12 +20,15 @@ PKGDIR = debian/storagenode
 
 override_dh_auto_install:
 # downloaded binaries directory
-	mkdir -p ${PKGDIR}/var/storagenode
+	mkdir -p ${PKGDIR}/var/lib/storagenode
+	mkdir -p ${PKGDIR}/etc/storagenode
 	dh_installsystemd --name=storagenode
 	dh_installsystemd --name=storagenode-updater
-	dh_installdebconf
-
 
+override_dh_fixperms:
+	dh_fixperms
+	chmod 755 ${PKGDIR}/var/lib/storagenode
```

Comment:
If we do need these directories to be created during package build, it might be easier to add `-m 755` do the `mkdir` commands in `override_dh_auto_install` rather than overriding dh_fixperms.

Reply:
I think permissions set by `auto_install` are overwritten by `dh_fixperms `

Reply:
Yeah, if we _don't_ need these directories to exist during package build, and we only care about them in the package, `dh_fixperms` alone should be fine.